ethportal-api/src/types/content_key/error.rs

```diff
@@ -12,3 +14,12 @@ pub enum ContentKeyError {
     #[error("Input Vec has length {received}, expected {expected})")]
     InvalidLength { received: usize, expected: usize },
 }
+
+impl ContentKeyError {
```

Comment:
I'm not great at rust, so this is just a bit of speculation and definitely a nitpick. Your instinct to create this fn is on point. But, I suspect that it's possible to do it even a bit more cleanly... I think if you `impl From<DecodeError> for ContentKeyError` then you might be able to do away with the `.map_err(..)` completely in `history.rs`. Maybe something like [this](https://stackoverflow.com/questions/57942901/how-to-implement-from-trait-for-custom-error-types)?

Reply:
I'm not sure I can do it exactly like that, because I'm not passing only `DecodeError`, I'm also passing `input` (`AsRef<[u8]`).

I guess I could still use `From` trait, but it would be from a tuple. Not sure if that is clean enough... What do you think?

Reply:
I wouldn't worry about it, it was just a thought, doesn't seem like it's worth more effort